Improved Responsiveness - Hidden elements in URL table on small screens

uxFeranmi opened the pull request:
## The Bug
On smaller screens, some of the share icons did not appear in the table.

## The Fix
1. Made the URL table horizontal scrolling so all content is accessible on all screen sizes.
2. Added a scroll indicator so the user is made aware of additional content to be scrolled into view.

*This PR closes #169*

uxFeranmi committed:
Merge pull request #40 from hngi/Develop

Syncing

uxFeranmi committed:
Merge pull request #41 from hngi/Develop

sync

uxFeranmi committed:
Horizontal scrolling for URLs table.

uxFeranmi committed:
Added scroll indicator to URLs table.

Stacalpha commented:
You may review the changes on your phone here. https://trim-app.herokuapp.com

It appears the scrolling doesn't work on an older version of Chrome for Android.

uxFeranmi committed:
Master (#43)

* Added cross-env as a dependency, not just devDependency, so it can be used in Heroku environment.

* jkfnv (#42)

* feat: implemeneted click metrics

* Create getMetrics.js

Mimddleware to get user's data gotten from the request object

* added timestamps to Click model

* Create metrics.js

Metric model created

* Update UrlShorten.js

Adding the metric model to each url

* Fix bug

Firefox form display not centre aligned
Cookies not working on Chrome 75+

* Improve click metrics

* Improve click metrics

* quick fix on social media sharing link

* Click metrics

* added db query for metrics

* Integrate click metrics frontend

* Fix chai test

* fix clipped url on frontend

* Update index.ejs

* Fix url undefined on production server

* Update about.ejs

Contact us link fixed, footer taken to partials

* Update footer.ejs

footer div brought to partials

* Update index.ejs

Advance form label changed, footer refactored

* Update footer.ejs

* Update index.ejs

link targets fixed

* Fix toggle button color and padding

* Update about.ejs

Logo link fixed

* Remove metrics from table

* s

* feat: add seo tags, closes #158

* Add support for older browsers

* Fix bug

* Update header.ejs

* fix: add correct metatag image

* About Page Redesign

* Contact Button

* Contact Button styles

* fix: change metatag image

* fix: use appropriate domain in metatags

* fix: fix copying

* Horizontal scrolling for URLs table.

* Added scroll indicator to URLs table.

* Fix table not scrolling.

* Bug fix. Trial 2.

* Fix bug. Trial 3.

* Revert bug fix trials.

* Fixed feedback button clash with table on small screen

uxFeranmi committed:
Merge branch 'Develop' into Develop
